packages/ember-application/lib/system/engine.js

```diff
@@ -16,18 +16,13 @@ import DefaultResolver from 'ember-application/system/resolver';
 import EngineInstance from './engine-instance';
 import isEnabled from 'ember-metal/features';
 import symbol from 'ember-metal/symbol';
-
 import Controller from 'ember-runtime/controllers/controller';
 import TextField from 'ember-templates/components/text_field';
```

Comment:
these should be done the same way, not use the ember-templates pseudo package.  Ideally all internal stuff would be done with injections and pseudo package just for the reexports